src/components/search/__tests__/search-test-cases.js

```diff
@@ -42,7 +43,7 @@ testCases.narrow = {
   element: <Narrow />
 };
 
-testCases.withConnector = {
+noRenderCases.withConnector = {
```

Comment:
💡 I switched this to a noRender test cases because it uses a dummy Swiftype site connector and will not return results